docker/mxnet-model-server.sh

```diff
@@ -99,6 +99,7 @@ start_mms()
     service nginx restart
     # Download and extract all the models
     python /mxnet_model_server/setup_mms.py $models
+    echo "0" >& /mxnet_model_server/.gpu_id
```

Comment:
I would recommend moving this line above setup_mms.py or moving it inside setup_mms.py. The guinicorn instances (created below) shouldn't be created unless the setup and this echo go through.